cmd/hub/cli/version.go

```diff
@@ -78,7 +77,7 @@ var versionCmd = &cobra.Command{
 			}
 		}
 
-		if version == "git" {
+		if version == "none" {
```

Comment:
Can we change `version` above to be `version := bi.GitSummary`?

Reply:
Then, no need to report "custom version". The version is what it is.

Reply:
just account for the 3rd case from go install. you could

* just print the GitSummary but leave the version check here. GitSummary will be "" when `go install ./...`. 
* or change the default in the buildinfo file so gitsummary default string is something other than "". but then will need to do some check still if you want to display more when the git info is available or remove that extra info so go install ./... version isn't weird empties